### PR TITLE
preparation for release of 7.0.0-alpha6

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| release of `7.0.0-alpha6` version 
|New feature| No 
|BC breaks| No 
|Fixes issues| ...
|Standards and tests pass| Yes but only locally. On CI it fails as `shopsys/microservice-product-search-export` has a dependency on `"shopsys/coding-standards": "7.0.0-alpha6"` that is not released yet
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docs/installation/installation-using-docker-linux.md [hunk 1]
```diff
@@ -9,16 +9,9 @@
...
+composer create-project shopsys/project-base --stability=alpha --no-install --keep-vcs
 cd project-base
 ```
-**Important note:** 
-
-**Due to changes in the installation process (see [changelog](/CHANGELOG.md) for details), we recommend using the current `dev-master` as a base for your project.**
-**Therefore there is the `stability=dev` flag used in the `composer` command above.**
-**After the next release, we will again return to our standard recommended stability (see notes below).**
-
-<!--- TODO Remove important note after tag release and change stability in composer line to standard -->
```
Comment (Miroslav-Stopka) [inline docs/installation/installation-using-docker-linux.md]: Najs :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1401,7 +1461,7 @@ That's why is this section formatted differently.
     - YesNoType
 - `.travis.yml` file with Travis CI configuration
 
-[Unreleased]: https://github.com/shopsys/shopsys/compare/v7.0.0-alpha5...HEAD
+[7.0.0-alpha6]: https://github.com/shopsys/shopsys/compare/v7.0.0-alpha5...v7.0.0-alpha6
```
[the remark this answers is not in the document]
Comment (vitek-rostislav) [reply CHANGELOG.md]: no no, we are not maintaining the "Unreleased" section anymore :wink:

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +28,66 @@ The changelog is generated during the release process using [ChangelogLinker](ht
...
+- [#463] warm up Symfony cache in microservices during start-up
+
+#### Changed
+
+- [#393] Continuous integration via Kubernetes
+    - for details, see [Introduction to Kubernetes](/docs/kubernetes/introduction-to-kubernetes.md)
+- [#444] Improve Postgres configuration to improve performance
+- [#430] Microservices are now built as Docker images
+- [#446] Generate CHANGELOG.md with ChangelogLinker, Thanks to [@TomasVotruba]
+- [#438] Attribute telephone moved from a billing address to the personal data…
```
[the remark this answers is not in the document]
Comment (vitek-rostislav) [reply CHANGELOG.md]: oh, I have missed that...thanks, fixed :+1:

File: project-base/docker/conf/docker-compose.yml.dist [hunk 4]
```diff
@@ -100,15 +100,15 @@ services:
...
+        image: shopsys/microservice-product-search:v7.0.0-alpha6
         container_name: shopsys-framework-microservice-product-search
         links:
             - elasticsearch
         depends_on:
             - elasticsearch
 
     microservice-product-search-export:
-        image: shopsys/microservice-product-search-export:latest
+        image: shopsys/microservice-product-search-export:7.0.0-alpha6
```
Comment (TomasVotruba) [inline project-base/docker/conf/docker-compose.yml.dist]: @rostislav-vitek Which one is correct, v7... or 7...

Here are used both :D

Comment (TomasVotruba) [reply project-base/docker/conf/docker-compose.yml.dist]: @vitek-rostislav Which one is correct, v7... or 7...

Here are used both :D

**Pull request merged**